test/cpp/jit/test_gpu.cpp

```diff
@@ -8593,7 +8593,7 @@ TEST(NVFuserTest, FusionSmemDynamicReductionSymbolic_CUDA) {
       __FILE__,
       "",
       lparams);
-  TORCH_CHECK(fe.kernel()->summary().war_hazard_syncs_count == 0);
+  // TORCH_CHECK(fe.kernel()->summary().war_hazard_syncs_count == 0);
```

Comment:
This test actually has a block reduction in a loop, so expect it to insert a war sync.